[2020-01-08 16:40:48,834] {taskinstance.py:630} INFO - Dependencies all met for <TaskInstance: trigger_dag.check_file_exists 2019-12-28T00:00:00+00:00 [queued]>
[2020-01-08 16:40:48,985] {taskinstance.py:630} INFO - Dependencies all met for <TaskInstance: trigger_dag.check_file_exists 2019-12-28T00:00:00+00:00 [queued]>
[2020-01-08 16:40:48,989] {taskinstance.py:841} INFO - 
--------------------------------------------------------------------------------
[2020-01-08 16:40:48,990] {taskinstance.py:842} INFO - Starting attempt 1 of 1
[2020-01-08 16:40:48,992] {taskinstance.py:843} INFO - 
--------------------------------------------------------------------------------
[2020-01-08 16:40:49,126] {taskinstance.py:862} INFO - Executing <Task(FileSensor): check_file_exists> on 2019-12-28T00:00:00+00:00
[2020-01-08 16:40:49,126] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'trigger_dag', 'check_file_exists', '2019-12-28T00:00:00+00:00', '--job_id', '36', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/trigger_dag.py', '--cfg_path', '/tmp/tmp71weecza']
[2020-01-08 16:40:51,359] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists /usr/local/lib/python3.7/site-packages/airflow/configuration.py:226: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2020-01-08 16:40:51,362] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   FutureWarning
[2020-01-08 16:40:51,363] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists /usr/local/lib/python3.7/site-packages/airflow/config_templates/airflow_local_settings.py:65: DeprecationWarning: The elasticsearch_host option in [elasticsearch] has been renamed to host - the old setting has been used, but please update your config.
[2020-01-08 16:40:51,363] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   ELASTICSEARCH_HOST = conf.get('elasticsearch', 'HOST')
[2020-01-08 16:40:51,364] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists /usr/local/lib/python3.7/site-packages/airflow/config_templates/airflow_local_settings.py:67: DeprecationWarning: The elasticsearch_log_id_template option in [elasticsearch] has been renamed to log_id_template - the old setting has been used, but please update your config.
[2020-01-08 16:40:51,365] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   ELASTICSEARCH_LOG_ID_TEMPLATE = conf.get('elasticsearch', 'LOG_ID_TEMPLATE')
[2020-01-08 16:40:51,365] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists /usr/local/lib/python3.7/site-packages/airflow/config_templates/airflow_local_settings.py:69: DeprecationWarning: The elasticsearch_end_of_log_mark option in [elasticsearch] has been renamed to end_of_log_mark - the old setting has been used, but please update your config.
[2020-01-08 16:40:51,366] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   ELASTICSEARCH_END_OF_LOG_MARK = conf.get('elasticsearch', 'END_OF_LOG_MARK')
[2020-01-08 16:40:52,105] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists [2020-01-08 16:40:52,105] {settings.py:252} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=644
[2020-01-08 16:40:54,001] {logging_mixin.py:112} INFO - [2020-01-08 16:40:53,978] {local_task_job.py:124} WARNING - Time since last heartbeat(0.20 s) < heartrate(5.0 s), sleeping for 4.797384 s
[2020-01-08 16:40:55,050] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists [2020-01-08 16:40:55,043] {__init__.py:51} INFO - Using executor CeleryExecutor
[2020-01-08 16:40:55,052] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists [2020-01-08 16:40:55,051] {dagbag.py:92} INFO - Filling up the DagBag from /usr/local/airflow/dags/trigger_dag.py
[2020-01-08 16:40:55,701] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists [2020-01-08 16:40:55,700] {cli.py:545} INFO - Running <TaskInstance: trigger_dag.check_file_exists 2019-12-28T00:00:00+00:00 [running]> on host eac77fa5e690
[2020-01-08 16:40:55,911] {taskinstance.py:1058} ERROR - The conn_id `fs_airflow` isn't defined
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 930, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 107, in execute
    while not self.poke(context):
  File "/usr/local/lib/python3.7/site-packages/airflow/contrib/sensors/file_sensor.py", line 57, in poke
    hook = FSHook(self.fs_conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/contrib/hooks/fs_hook.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
    conns = cls._get_connections_from_db(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
    "The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `fs_airflow` isn't defined
[2020-01-08 16:40:55,921] {taskinstance.py:1089} INFO - Marking task as FAILED.
[2020-01-08 16:40:56,029] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists Traceback (most recent call last):
[2020-01-08 16:40:56,029] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/bin/airflow", line 37, in <module>
[2020-01-08 16:40:56,030] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     args.func(args)
[2020-01-08 16:40:56,031] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2020-01-08 16:40:56,031] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     return f(*args, **kwargs)
[2020-01-08 16:40:56,032] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 551, in run
[2020-01-08 16:40:56,033] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     _run(args, dag, ti)
[2020-01-08 16:40:56,034] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 469, in _run
[2020-01-08 16:40:56,036] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     pool=args.pool,
[2020-01-08 16:40:56,038] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-01-08 16:40:56,040] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     return func(*args, **kwargs)
[2020-01-08 16:40:56,044] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 930, in _run_raw_task
[2020-01-08 16:40:56,044] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     result = task_copy.execute(context=context)
[2020-01-08 16:40:56,054] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 107, in execute
[2020-01-08 16:40:56,055] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     while not self.poke(context):
[2020-01-08 16:40:56,057] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/contrib/sensors/file_sensor.py", line 57, in poke
[2020-01-08 16:40:56,057] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     hook = FSHook(self.fs_conn_id)
[2020-01-08 16:40:56,058] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/contrib/hooks/fs_hook.py", line 38, in __init__
[2020-01-08 16:40:56,058] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     conn = self.get_connection(conn_id)
[2020-01-08 16:40:56,060] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
[2020-01-08 16:40:56,061] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     conn = random.choice(list(cls.get_connections(conn_id)))
[2020-01-08 16:40:56,062] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
[2020-01-08 16:40:56,064] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     conns = cls._get_connections_from_db(conn_id)
[2020-01-08 16:40:56,066] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-01-08 16:40:56,067] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     return func(*args, **kwargs)
[2020-01-08 16:40:56,070] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
[2020-01-08 16:40:56,075] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists     "The conn_id `{0}` isn't defined".format(conn_id))
[2020-01-08 16:40:56,076] {base_task_runner.py:115} INFO - Job 36: Subtask check_file_exists airflow.exceptions.AirflowException: The conn_id `fs_airflow` isn't defined
[2020-01-08 16:40:58,815] {logging_mixin.py:112} INFO - [2020-01-08 16:40:58,811] {local_task_job.py:103} INFO - Task exited with return code 1
